diktat-rules/src/test/kotlin/com/saveourtool/diktat/ruleset/chapter3/FileStructureRuleFixTest.kt

```diff
@@ -96,4 +100,50 @@ class FileStructureRuleFixTest : FixTestBase("test/paragraph3/file_structure", :
     fun `invalid move in kts files`() {
         fixAndCompare("ScriptPackageDirectiveExpected.kts", "ScriptPackageDirectiveTest.kts")
     }
+
+    @Test
+    @Tag(WarningNames.FILE_NO_BLANK_LINE_BETWEEN_BLOCKS)
+    fun `several empty lines after package`(@TempDir tempDir: Path) {
+        val folder = tempDir.resolve("src/main/kotlin/com/saveourtool/diktat").also {
+            it.createDirectories()
+        }
+        val testFile = folder.resolve("test.kt").also {
+            it.writeText("""
+                package com.saveourtool.diktat
+                /**
+                 * @param bar
+                 * @return something
+                 */
+                fun foo1(bar: Bar): Baz {
+                    // placeholder
+                }
+            """.trimIndent())
+        }
+        val expectedFile = folder.resolve("expected.kt").also {
+            it.writeText("""
+                package com.saveourtool.diktat
+                /**
+                 * @param bar
+                 * @return something
+                 */
+                fun foo1(bar: Bar): Baz {
+                    // placeholder
+                }
+            """.trimIndent())
+        }
+        fixAndCompare(
+            expectedFile,
+            testFile,
+            overrideRulesConfigList = listOf(
+                RulesConfig(
+                    name = WarningNames.FILE_NO_BLANK_LINE_BETWEEN_BLOCKS,
```

Comment:
looks like one of `FILE_NO_BLANK_LINE_BETWEEN_BLOCKS` or `WRONG_NEWLINES_AROUND_KDOC` generates two new lines when package is missied. 
Need to analyze PACKAGE_NAME_MISSING -- looks like it puts package in invalid places

Reply:
the problem was in empty import list -- PACKAGE rule adds the WHITE_SPACE after package, but FILE_STRUCTURE rule adds it after IMPORT_LIST